java/src/jmri/jmrix/tams/TamsSensorManager.java

```diff
@@ -10,10 +14,9 @@
 /**
  * Implement sensor manager for Tams systems. The Manager handles all the state\
  * changes Requires v1.4.7 or higher of TAMS software to work correctly
- * <p>
- * System names are "TSnnn:yy", where T is the user configurable system prefix,
- * nnn is the Tams Object Number for a given S88 Bus Module and
- * yy is the port on that module.
+ * <P>
+ * System names are "TMSnnn:yy", where nnn is the Tams Object Number for a given
```

Comment:
Is this simply a fix to the comments? Or does the code-change make a change to the system name?